### ISSUE TITLE
[Firefox] Screen reader does not identify the telemetry toggle button as a toggle on welcome page

### ISSUE THREAD

**Issue by acornestean** [mozilla-extensions/firefox-bridge#67]
**Description:**
Screen reader does not identify the telemetry opt in/out toggle as a toggle. Only the string is announced.

**Preconditions:**
1. A screen reader is enabled in the OS

**Steps to reproduce:**
1. Load the Firefox Firefox Bridge v0.1 in the browser
2. The welcome page opens in a new tab and the screen reader start announcing the page content
3. Notice the screen reader does not announce the toggle button as toggle and only reads the associated string

**Expected:**
The screen reader should announce the toggle button as toggle.

**Actual:**
The screen reader does not announce the toggle button as toggle and only reads the associated string.

**Comment by acornestean** [comment]
Verified as Fixed with Alpha Build V2 extension.

Screen reader identifies toggle as “checkbox” stating whether it’s checked or unchecked.
